src/components/Profile.tsx

```diff
@@ -6,10 +6,15 @@ import { useAppSelector } from "../redux/hooks";
 import * as types from "../utilities/types";
 import { saveProfile } from "../services/content";
 import { core } from "@dsnp/sdk";
+import GraphChangeButton from "./GraphChangeButton";
 
 const Profile = (): JSX.Element => {
   const userId: string | undefined = useAppSelector((state) => state.user.id);
 
+  const following = useAppSelector(
```

Comment:
for the sake of continuity with connectionsListProfiles, should we use `userImFollowing` or `userFollowing`?

Reply:
This is actually, the other map of the users the displayed user is following. I guess `followedByDisplayedUser`? I'm not crazy about the passive voice. `displayUserFollowingProfiles`? I still think the passive voice is clearer.